system/ee/ExpressionEngine/Library/CP/FileManager/Traits/FileManagerTrait.php

```diff
@@ -126,7 +126,11 @@ protected function listingsPage($uploadLocation = null, $view_type = 'list', $fi
         $search_terms = ee()->input->get_post('filter_by_keyword');
 
         if ($search_terms) {
-            $files->search(['title', 'file_name', 'mime_type'], $search_terms);
+            if (is_numeric($search_terms) && strlen($search_terms) < 3) {
```

Comment:
Does this mean that a file_id of 3 or more digits can't be searched? E.g. I can search for file #23, but not #456?

Reply:
That's interesting, the `search()` on line 132 should be looking in the file_id field as well now.  But I just did a similar test and got the same result as you @litzinger.  This needs another look.

I think I would also prefer doing a more performant `filter()` when the search term looks like `id:12345`.  But regardless we should be searching inside the `file_id` along with the other fields as this original change intended.